docs/css/mkdocstrings.css

```diff
@@ -1,9 +0,0 @@
-.md-header__topic {
-    font-size: 200%;
-    font-family: Verdana, Geneva, Tahoma, sans-serif;
-}
-
-.md-header__button.md-logo img {
-    height: 22%;
-    width: 22%;
-  }
```

Comment:
i was scrolling the docs today and didn't super-like the big header...I think the default are good enough